docs/FATE_Upgrade.md

```diff
@@ -0,0 +1,284 @@
+nfsServer myshareddir → ls
+fate-9999-client-data-pvc-e7ea7c79-c5e3-4824-8269-4eac390dcae8         fate-9999-nodemanager-1-data-pvc-eed13a83-442e-4c9d-bb00-e6a7ce7c05f2
+fate-9999-mysql-data-pvc-0865a743-9bfb-4990-bd26-84b0241f0084          fate-9999-python-data-pvc-5fc77a70-fe67-4b10-8db5-d3b0ef76d8f7
+fate-9999-nodemanager-0-data-pvc-eaac241e-4d0f-4d06-96d0-30e85e284adc
+nfsServer myshareddir → ls fate-9999-mysql-data-pvc-0865a743-9bfb-4990-bd26-84b0241f0084 | grep snapshot
+snapshot.sql
+```
+
+### Using KubeFATE CLI to do the upgrade
+We just need to change two lines of the cluster.yaml file:
+```
+chartVersion: v1.7.2	  ->   chartVersion: v1.8.0
+imageTag: 1.7.2-release   ->   imageTag: 1.8.0-release
+```
+Then execute:
+```
+kubefate cluster update -f cluster.yaml
+```
+A KubeFATE job will be created. At this time, a K8s one-time job will be launched, it will do 2 things:
+1. Shut down the "python" pod, which includes FATE-Flow and FATE-Board.
+2. It executes one or more .sql script(s) to help upgrade the schema of the MySQL database.
+
+After that, the K8s job will be deleted by default. For debugging purpose, to keep the finished job on the K8s cluster, we can run:
+```
+kubefate cluster update -f cluster.yaml --keepUpgradeJob
+```
+Soon, the new pods will be spawned:
+```
+fate-9999       client-7dd4fc8cbf-jtz2x                                   1/1     Running     0          5m4s
+fate-9999       clustermanager-7987759b84-7bgbf                           1/1     Running     0          5m26s
+fate-9999       mysql-fc76db6db-rdvw5                                     1/1     Running     0          5m26s
+fate-9999       nodemanager-0-54845ccd98-jjptt                            2/2     Running     0          5m25s
+fate-9999       nodemanager-1-cdd975587-cn96q                             2/2     Running     0          5m26s
+fate-9999       python-57f8c857d8-ckp2z                                   2/2     Running     0          5m36s
+fate-9999       rollsite-6b67d75c78-bngqb                                 1/1     Running     0          5m26s
+```
+At this time, the cluster has already been upgrade to v1.8.0, which means the upgrade has been done:
+```
+➜ kubefate cluster list                                                        
+UUID                                    NAME            NAMESPACE       REVISION        STATUS  CHART   ChartVERSION    AGE  
+4a8372d1-a15e-4653-8420-60627d89f858    fate-9999       fate-9999       2               Running fate    v1.8.0          3h35m
+```
+
+## Workarounds
+
+As mentioned, due to a historical reason, upgrading from a FATE cluster whose version is less or equal than v1.7.1 is not supported by KubeFATE. However, there is a manual workaround for that purpose:
+
+Suppose we have a v1.7.1 FATE cluster, how to upgrade to v1.7.2 manually without data loss in the MySQL database?
+### Export the data in the MySQL database
+Get into the container of MySQL, go to "/var/lib/mysql" and run below command:
```

Comment:
Should we point to the lines we showed above how we getting into the mysql container?